File: task-03/Python/subtask-2.py
def file_read_and_write():
    file_one = open("/home/aakash/amfoss-tasks/task-03/Python/input.txt", "r")
    data = file_one.read()
    file_one.close()
    
    file_two = open("/home/aakash/amfoss-tasks/task-03/Python/output.txt","w")
    file_two.write(data)
    
    file_two.close()
file_read_and_write()

File: task-03/C/subtask-2.c
#include <stdio.h>
#include <stdlib.h>

void fileReadAndWrite() {
    FILE *fileReader = NULL;
    FILE *fileWriter = NULL;
    char *inputPath = "/home/aakash/amfoss-tasks/task-03/C/input.txt";
    char *outputPath = "/home/aakash/amfoss-tasks/task-03/C/output.txt";
    
    fileReader = fopen(inputPath, "r");
    if (fileReader == NULL) {
        perror("Error opening input file");
        return;
    }

    fileWriter = fopen(outputPath, "w");
    if (fileWriter == NULL) {
        perror("Error opening output file");
        fclose(fileReader);
        return;
    }

    int character;
    while ((character = fgetc(fileReader)) != EOF) {
        fputc(character, fileWriter);
    }

    fclose(fileReader);
    fclose(fileWriter);
}

int main() {
    fileReadAndWrite();
    return 0;
}
